app/app_main.c

```diff
@@ -1954,7 +1995,8 @@ static ACVP_RESULT app_rsa_handler(ACVP_TEST_CASE *test_case)
 }
 
 
-#ifdef ACVP_NO_RUNTIME
+// #ifdef ACVP_NO_RUNTIME
```

Comment:
I thought I had cleaned up everything I messed with while developing... guess not! Fixed and will push it up :)